src/sempy_labs/lakehouse/_get_lakehouse_tables.py

```
@@ -83,45 +86,85 @@ def get_lakehouse_tables(
-        uses_pagination=True,
-        client="fabric_sp",
-    )
+    API_called = True
+    try:
+        responses = _base_api(
+            request=f"v1/workspaces/{workspace_id}/lakehouses/{lakehouse_id}/tables",
+            uses_pagination=True,
+            client="fabric_sp",
+        )
+
+    except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)